src/resources/views/crud/filters/select2_multiple.blade.php

```diff
@@ -86,11 +86,11 @@ class="nav-item dropdown {{ Request::get($filter->name)?'active':'' }}">
 
                 $(this).change(function() {
 	                var value = '';
-	                if ($(this).val() !== null) {
+	                if (Array.isArray($(this).val())) {
 	                    // clean array from undefined, null, "".
-	                    var values = $(this).val().filter(function(e){ return e === 0 || e });
+	                    var values = $(this).val().filter(Boolean);
```

Comment:
I love this `.filter(Boolean);` 👌
But if we previously had `e === 0 || e`, probably there were situation where the value could be `0`.

To avoid a Breaking Change we should stick with the previous code;
```suggestion
	                    var values = $(this).val().filter(function(e){ return e === 0 || e });
```